sklearn/utils/estimator_checks.py

```diff
@@ -156,6 +157,30 @@ def check_supervised_y_no_nan(name, estimator_orig):
                          "array y with NaN value.".format(name))
 
 
+@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+def check_constant_features(name, estimator_orig):
+    # Checks that estimators work with constant features
+    # or raise a reasonable error message
+
+    # create features with a constant column
+    rng = np.random.RandomState(888)
+    X = np.abs(rng.randn(10, 5))
+    X[:, 1] = 1
+    X = pairwise_estimator_convert_X(X, estimator_orig)
+    # create target
+    y = np.arange(10) % 2
+    estimator = clone(estimator_orig)
+    y = enforce_estimator_tags_y(estimator, y)
+
+    try:
+        estimator.fit(X, y)
+    except FloatingPointError as e:
```

Comment:
Not sure `FloatingPointError` and `errmsg`  are  general enough for this. RuntimeError (or ValueError) would be more appropriate.

I understand you used this because of `GraphicalLasso`. Maybe we could change the raised exception there instead.